app/src/main/java/org/walleth/activities/MainActivity.kt

```diff
@@ -319,3 +326,13 @@ class MainActivity : AppCompatActivity(), SharedPreferences.OnSharedPreferenceCh
         super.onDestroy()
     }
 }
+
+fun Activity.copyToClipboard(address: Address, view: View) {
+    copyToClipboard("ethereum:${address.hex}", view)
+}
+
+fun Activity.copyToClipboard(ethereumString: String, view: View) {
```

Comment:
I think this should live outside MainActivity.kt

Reply:
What is a good place?

Reply:
I would say util/Clipboard.kt